server.js

```diff
@@ -1,12 +1,23 @@
 const express = require('express');
+const server = express();
```

Comment:
It is a bit strange to have a variable defined before you have finished importing all your dependencies. It's not wrong per say, just strange.